feature-detects/mediastream.js

```diff
@@ -0,0 +1,21 @@
+/*!
```

Comment:
I mean, at this point we should just rename the file to `mediarecorder.js` as having different names is just going to confuse anyone who is looking at it in the future. At least those are my thoughts, however if you want to maintain the name, I leave the decision up to you (not that the world will end for the name of this file, I'm sure 2020 has something better to end the world with 😅)

Don't forget to change it in `config-all.json` too if you change it :)